spotify_player/src/event/mod.rs

```diff
@@ -240,11 +228,7 @@ pub fn handle_action_in_context(
                 })?;
             }
             Action::ShowActionsOnArtist => {
-                ui.popup = Some(PopupState::ArtistList(
-                    ArtistPopupAction::ShowActions,
-                    album.artists,
-                    new_list_state(),
-                ));
+                self::handle_show_actions_on_artist(album.artists, data, ui);
```

Comment:
```suggestion
                handle_show_actions_on_artist(album.artists, data, ui);
```

Reply:
Done. I wonder why CI doesn't detect it?